Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "osmpbfreader"
-version = "0.17.0"
+version = "0.17.1"
```

As there is breaking change, minor must be updated
```suggestion
version = "0.18.0"
```